doc/source/appendices/tutorial.rst

```diff
@@ -23,199 +23,126 @@ Notes:
   * CICE and Icepack Resources, https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index
 
 
-Github One-time Configuration
-----------------------------------
+Create An Icepack Branch, Port, and Run Initial Case
+------------------------------------------------------
 
-You need to have your own GitHub account before you can start the following activities, and you should have already forked the Icepack repository.
-For information about how to set up a GitHub account for the Icepack repository, see the Consortium documentation here, https://github.com/CICE-Consortium/About-Us/wiki/Git-Workflow-Guide.  The Consortium recommends that you keep your fork’s main branch in sync with the Consortium version and that you always work on branches.  This is all documented in the Git-Workflow-Guide linked above. 
+You should fork the Icepack repository and create a new branch in your fork for development.  This is documented in https://github.com/CICE-Consortium/About-Us/wiki/Git-Workflow-Guide.  But basically,
 
-Note: 
+* Create a github account for yourself
+* Fork the Consortium Icepack repository using the "Fork" feature at https://github.com/CICE-Consortium/Icepack
+* Clone your repository onto your machine
+* Create a branch
+* Checkout the branch
+* Port the model and verify you can build and run the model
 
-* The workflow guide is oriented toward setting up CICE rather than Icepack, but the same workflow applies to Icepack standalone.  Icepack can be set up and run as an independent model following the same workflow.
+You may be able to leverage the conda port to build and run the model, you may be working on a supported machine, or you may have to port the model to your machine.  Porting, setting up cases, building, running, and running test suites are all documented in the Icepack User Guide, https://cice-consortium-icepack.readthedocs.io/en/latest/index.html.  There are many ways to setup and run the model.
 
+**Note:** The workflow guide is oriented toward setting up CICE rather than Icepack, but the same workflow applies to Icepack standalone.
 
-Clone Icepack
--------------------
+To summarize the steps in greater details **assuming use of conda in a Mac or Linux environment**.
```

Comment:
Note that these steps only need to be done once.
Remove the s from 'details'.

Reply:
I added some words in a few places about "done once per user" or "done once per machine".